### PR TITLE
Add ent binaries and toggle via env variable

### PR DESCRIPTION
**Pull request by zhenik**
Approach with a symlink.
1. [install phase] Download oss(open source) & ent(enterprise) binaries in different directories
2. [bootstrap phase] Create symlink(s) to `/usr/local/bin/<binary>` to `oss` or `ent` pathes depends on env variable `<daemon>_enterprise`

### CODE REVIEW

**Comment by zhenik** [comment]
@Neha-Sinha2305 Where do I need a test, when `consul_enterprise=true`?

File: ansible/group_vars/all/variables.yml [hunk 1]
```diff
@@ -10,6 +10,10 @@ hashicorp:
     vault: 1.5.0
     nomad: 0.12.2
     packer: 1.6.1
+  tools_enterprise:
+    consul: 1.8.3+ent
+    nomad: 0.12.3+ent
+    vault: 1.5.0+ent
 ansible:
```
Comment (Neha-Sinha2305) [inline ansible/group_vars/all/variables.yml:17]: What about terraform enterprise version?

Comment (fredrikhgrelland) [reply ansible/group_vars/all/variables.yml:17]: It is a cloud solution and does not apply ;)

File: ansible/templates/.env_default.j2 [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {% raw -%}
 #Control box features
+consul_enterprise=false
```
Comment (zhenik) [inline ansible/templates/.env_default.j2:3]: Separate toggle via 3 env variables:
```
consul_enterprise=false
nomad_enterprise=false
vault_enterprise=false
```

**Comment by Neha-Sinha2305** [comment]
The tests have run fine and it installed the open source version for hashistack which is the default behavior. Should we expand the github action matrix to include enterprise version installation as part of this PR or should i write a new issue and work on that separately?

**Comment by zhenik** [comment]
> The tests have run fine and it installed the open source version for hashistack which is the default behavior. Should we expand the github action matrix to include enterprise version installation as part of this PR or should i write a new issue and work on that separately?

`install.sh` install both `oss` and `ent` versions. Later in `bootstrap.sh` toggle for which version to run, controlled via symlink. I guess that test(s) should check toggle, not the installation.

**Comment by Neha-Sinha2305** [comment]
True.. that! However, we need to add the new variables in github action to override the default values for them and that would verify the toggle.

**Comment by fredrikhgrelland** [comment]
> True.. that! However, we need to add the new variables in github action to override the default values for them and that would verify the toggle.

Follow up in #197

**Pull request merged**